hw09-aop/src/main/java/ru/otus/merets/exception/CreateWatcherException.java

```diff
@@ -0,0 +1,7 @@
+package ru.otus.merets.exception;
+
+public class CreateWatcherException extends Exception {
```

Comment:
```suggestion
public class CreateWatcherException extends RuntimeException {
```

Смысла в проверяемом исключении нет. От них все уходят постепенно.